recipe/meta.yaml

```diff
@@ -34,35 +31,42 @@ requirements:
     - {{ compiler('cxx') }}
     # llvm is needed for the headers
     - llvm-openmp              # [osx]
-    - patch     # [not win]
-    - m2-patch  # [win]
+
   host:
     - python
     - pip
     - setuptools
     - wheel
-    - llvmlite 0.38.*
+    - llvmlite 0.39.*
```

Comment:
Are pip and wheel required? I am not sure, but it likely doesn't hurt to leave them here.

Reply:
I'm not sure either, but I agree it's fine as-is.